### ISSUE TITLE
Running docker image on RPi throws the 'exec user process caused "exec format error" '

### ISSUE THREAD

**Issue by the100rabh** [n8n-io/n8n#600]
**Describe the bug**
Running the docker image with the standard commands fails.

**To Reproduce**
1. run the command from command line 
`docker run -it --rm   --name n8n   -p 5678:5678   -v ~/.n8n:/root/.n8n   n8nio/n8n`
2. See error
```
standard_init_linux.go:211: exec user process caused "exec format error"
failed to resize tty, using default size

```
**Expected behavior**
Should run the n8n docker image correctly 

**Environment (please complete the following information):**
 - OS: Raspbian Stretch
- Docker Version:           19.03.9

**Comment by janober** [comment]
Ah yes for a Raspberry Pi some things have to be done differently and the default image will not work. One of our amazing community members just created something to get n8n easily up and running. You can find it here:
https://community.n8n.io/t/n8n-pre-built-raspberry-pi-image-available-n8n-pi/1437
and here:
http://n8n-pi.tephlon.xyz/#/README

**Comment by janober** [comment]
Guess you found what you are looking for. So close this issue now.

**Comment by RyanPnayR** [comment]
I was able to get the docker image to build on a 2gb raspberry pi 4 running Raspbian 10 by changing the base image to use arm32v7/node:12.16 instead of node:12.16. It seems to be working I have not run into any kinks with it yet.

**Comment by the100rabh** [comment]
@janober I dont think this is even a useful path, flashing just to run n8n on RPi is highly inefficient use of RPi. 

@RyanPnayR Could you share your dockerimage on dockerhub

**Comment by RyanPnayR** [comment]
@the100rabh I created a repo for it [here](https://github.com/RyanPnayR/n8n-docker-rpil)

**Comment by janober** [comment]
I now finally found some time to look into it further. Can you please try if the following docker image works fine for you: `n8nio/n8n:0.70.0-rpi`

If it does will I make sure that it will be automatically built with each version release.

**Comment by janober** [comment]
Did sadly not hear anything back for 2 weeks. Did anybody test it?

**Comment by RyanPnayR** [comment]
@janober That image you shared is working for me.

**Comment by janober** [comment]
Great. Thanks a lot. Will then make sure that it will automatically be built for all future releases.

**Comment by janober** [comment]
Ok did set it up. Should now hopefully always built it. We will see with the next version if everything works fine.

**Comment by janober** [comment]
Ok we just released n8n@0.73.0 and it seems like the automatic built of the RPi version worked fine. Should be available as:
`n8n:0.73.0-rpi`
